doc/radosgw/lua-scripting.rst

```diff
@@ -135,6 +135,12 @@ Debug Log
 The ``RGWDebugLog()`` function accepts a string and prints it to the debug log with priority 20.
 Each log message is prefixed ``Lua INFO:``. This function has no return value.
 
+The ``RGWDebugLog2()`` function accepts a format string and a variable list of arguments,
```

Comment:
how about `RGWDebugLogF()` or `RGWFormatDebugLog()` or something similar?

Reply:
I have changed the name to RGWDebugLogF.